test/extensions/common/wasm/wasm_runtime.cc

```diff
@@ -28,29 +22,35 @@ std::vector<std::string> sandboxRuntimes() {
   return runtimes;
 }
 
-std::vector<std::string> languages() {
+std::vector<std::string> languages(bool cpp_only) {
   std::vector<std::string> languages;
-#if !defined(__aarch64__)
-  // TODO(PiotrSikora): There are no Emscripten releases for arm64.
+#if !!defined(__x86_64__)
```

Comment:
Do we really need this double negation?

Reply:
Good catch, it's not. It's a result of mass find-and-replace (`defined(__aarch64__)` -> `!defined(__x86_64__)`).

But this is actually an issue in the underlying PR (#19615).